### ISSUE TITLE
Accessibility: The user is unable to toggle between shipping methods using the keyboard

### ISSUE THREAD

**Issue by AndrewBarber** [bigcommerce/checkout-js#1725]
On checkout the user should be able to hit the tab key to navigate between different shipping methods and then hit the enter key to select any of the highlighted shipping methods.

However, if the user hits the tab key on shipping methods this will skip past any shipping methods displayed after the first one. Additionally if the user presses the enter key on the already selected shipping method, this will progress the user to the Billing / Payment step.

This will stop the user from selecting a desired shipping method if they are limited to using a keyboard to navigate checkout.

The video below shows the issue where the shipping method is selected, and hitting the enter key skips the user to the payment step.
